patches/net/minecraft/core/MappedRegistry.java.patch

```diff
@@ -17,6 +26,12 @@
        this.validateWrite(p_256594_);
        Validate.notNull(p_256594_);
        Validate.notNull(p_256374_);
++      if (p_256563_ > this.getMaxId())
```

Comment:
Interesting - I wonder what the max ID is used for.

Reply:
See: custom display contexts.